### PR TITLE
Add type definitions for merge-refs

### PR DESCRIPTION
**Pull request by Skona27**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`

### CODE REVIEW

File: types/merge-refs/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// Type definitions for merge-refs 1.0
+// Project: https://github.com/wojtekmaj/merge-refs
+// Definitions by: Jakub Skoneczny <https://github.com/Skona27>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+
+declare function merge_refs<T>(...refs: Array<React.Ref<T>>): (instance: T) => void;
+export = merge_refs;
```
Comment (sheetalkamat) [inline types/merge-refs/index.d.ts:9]: This needs to be exported using `default` instead.
https://unpkg.com/browse/merge-refs@1.0.0/dist/umd/index.js

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/merge-refs/index.d.ts (hunk 1 above)
Comment (Skona27) [reply types/merge-refs/index.d.ts]: Changed this to be `export default`

**Review by RyanCavanaugh — APPROVED** [review]

**Comment by Skona27** [comment]
Ready to merge

**Comment by wojtekmaj** [comment]
FYI, this is no longer needed as merge-refs is now natively in TypeScript :)

**Pull request merged**